api/src/main/resources/workbench.yaml

```diff
@@ -2098,33 +2125,55 @@ definitions:
       sortColumn:
         type: string
 
-  ParticipantCohortStatusesRequest:
-    type: object
+  PageFilterRequest:
+    discriminator: pageFilterType
     required:
       - page
       - pageSize
-      - sortColumn
       - sortOrder
+      - pageFilterType # required for inheritance to work
     properties:
       page:
         type: integer
         description: specific page (default is 0)
       pageSize:
         type: integer
         description: page size of results (default is 25)
-      sortColumn:
-        $ref: "#/definitions/ParticipantCohortStatusColumns"
-        description: Column upon which to sort (default is 'participantId')
       sortOrder:
         $ref: "#/definitions/SortOrder"
-        description: Sort direction (default is ascending)
+      pageFilterType:
+        $ref: "#/definitions/PageFilterType"
       filters:
```

Comment:
What are examples of the filters that can be provided here?

(I'm curious to see how easy it will be to use the materialize cohort endpoint for this in future...)

Reply:
Chatted offline. You'll be able to filter on any column returned.